src/internal/components/chart-popover/index.tsx

```diff
@@ -14,13 +14,15 @@ import { nodeBelongs } from '../../utils/node-belongs';
 
 import popoverStyles from '../../../popover/styles.css.js';
 import styles from './styles.css.js';
+import testClasses from './test-classes/styles.css.js';
 
-interface ChartPopoverProps extends PopoverProps {
+export interface ChartPopoverProps extends PopoverProps {
   /** Title of the popover */
   title?: React.ReactNode;
 
   /** References the element the container is positioned against. */
-  trackRef: React.RefObject<HTMLElement | SVGElement>;
+  trackRef?: React.RefObject<HTMLElement | SVGElement>;
+  getTrack?: () => null | HTMLElement | SVGElement;
```

Comment:
getTrack is an alternative for trackRef. The difference is that it is called every time the track element is requested, whilst the trackRef can get stale.